### PR TITLE
Added value constraints to GDP traffic split and edited gslb.go to pr…

### PR DESCRIPTION
**Pull request by RiyaGaniga-avi**
1. Added value constraints to GDP traffic split.
2. Edited gslb.go to print the error

### CODE REVIEW

File: gslb/ingestion/gslb.go [hunk 1]
```diff
@@ -719,7 +719,7 @@ func InformersToRegister(oclient *oshiftclient.Clientset, kclient *kubernetes.Cl
 	allInformers := []string{}
 	_, err := kclient.CoreV1().Services("").List(metav1.ListOptions{TimeoutSeconds: &informerTimeout})
 	if err != nil {
-		gslbutils.Errf("cname: %s, cluster api server health check failed, can't access the services api", cname)
+		gslbutils.Errf("cname: %s, cluster api server health check failed, can't access the services api as %s", cname, err)
```
Comment (hemantavi) [inline gslb/ingestion/gslb.go:722]: Can we rephrase this to:
```
can't access the services api, error: %s
```

**Review by hemantavi — COMMENTED** [review]
LGTM with one minor change

File: gslb/ingestion/gslb.go (hunk 1 above)
Comment (RiyaGaniga-avi) [reply gslb/ingestion/gslb.go]: Done!

**Review by hemantavi — APPROVED** [review]

**Review by monotosh-avi — APPROVED** [review]
